reports/Google-2018-report.md

```diff
@@ -13,13 +15,18 @@ prioritize transparency, diversity, and reproducibility to foster collaboration.
 
+In neuroscience, data such as magnetic resonance images and physiology readings can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but incompatible specifications cause portability issues between different labs and scientists. The brain imaging data standard (BIDS) is a framework for organizing data that standardizes datasets across different researchers to enable easier sharing.
 
 
-In neuroscience, data such as magnetic resonance images and physiology readings can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but incompatible specifications cause portability issues between different labs and scientists. The brain imaging data standard (BIDS) is a framework for organizing data that standardizes datasets across different researchers to enable easier sharing.
+## My role
 
-## Our role
+The primary goal of this project was to simplify the process of learning about BIDS and engage more users.
+Well before it was funded by Google, the roots of this initiative were being pioneered by a small group of people around the world that were acting as local BIDS ambassadors in their communities.
+Many of us were already giving presentations at every workshop and conference possible.
+However, these were only for a limited audience, and there was a need for central organization.
+We hoped to create a central repository accessible to everyone that would contain easy to read documentation and a collection of useful resources. 
 
-The primary goal of this project was to simplify the process of learning about BIDS and engage more users. Well before it was funded by Google, the roots of this initiative were being pioneered by a small group of people around the world that were acting as local BIDS ambassadors in their communities. Many of us were already giving presentations at every workshop and conference possible. However, these were only for a limited audience, and there was a need for central organization. We hoped to create a central repository accessible to everyone that would contain easy to read documentation and a collection of useful resources. 
+It way my responsibility over the Google Summer of Code to be the lead developer and maintainer of the repository. When responding to requests from both senior developers and new contributors alike, this meant breaking down requested features into action items that could be assigned to either myself or the relevant expert. Periodically, I would then go through the list of issues to make sure progress on them “die out”. As a content creator, my work was primarily focused on the README and wiki. This included writing content and enhancing user experience by designing a clear framework. 
```

Comment:
"...to make sure progress on them die out" ? Unclear what was meant, here.

Reply:
I've rephrased this to make it clearer (I hope that the goal is to *not* let the issues "die out" 😄